File: CodeChef/Self Learning/Getting Started/Id_and_Ship.py
# CodeChef - FLOW010 - Id and Ship
# https://www.codechef.com/LP0TO101/problems/FLOW010

import sys
input = sys.stdin.readline

T = int(input())
for _ in range(T):
    id_char = input().rstrip().lower()

    ship_class = None
    if id_char == 'b':
        ship_class = "BattleShip"
    elif id_char == 'c':
        ship_class = "Cruiser"
    elif id_char == 'd':
        ship_class = "Destroyer"
    elif id_char == 'f':
        ship_class = "Frigate"
    
    print(ship_class)
    

File: CodeChef/Self Learning/Getting Started/Id_and_Ship.cpp
// CodeChef - FLOW010 - Id and Ship
// https://www.codechef.com/LP0TO101/problems/FLOW010

#include "bits/stdc++.h"
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int T;
    cin >> T;
    char id;
    for(int i = 1; i <= T; ++i){
        cin >> id;
        id = tolower(id);
        string ship_id;
        switch(id) {
            case 'b':
                ship_id = "BattleShip";
                break;
            case 'c':
                ship_id = "Cruiser";
                break;
            case 'd':
                ship_id = "Destroyer";
                break;
            case 'f':
                ship_id = "Frigate";
                break;
        }

        cout << ship_id << endl;
    }

}
